### PR TITLE
chore: update RestAssured usage

### PR DESCRIPTION
**Pull request by aleboulanger**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4961

**Please check if the PR fulfills these requirements**
- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/203/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/203/)

File: dataprep-api/src/test/java/org/talend/dataprep/helper/OSDataPrepAPIHelper.java [hunk 1]
```diff
@@ -588,8 +559,7 @@ protected AsyncExecutionMessage waitForAsyncMethodToFinish(String asyncMethodSta
                     mapper.readerFor(AsyncExecutionMessage.class).readValue(statusAsyncMethod);
 
             AsyncExecution.Status asyncStatus = asyncExecutionMessage.getStatus();
-            isAsyncMethodRunning =
-                    asyncStatus.equals(AsyncExecution.Status.RUNNING) || asyncStatus.equals(AsyncExecution.Status.NEW);
+            isAsyncMethodRunning = asyncStatus == RUNNING || asyncStatus == NEW;
```
Comment (tlnd-jbrun) [inline dataprep-api/src/test/java/org/talend/dataprep/helper/OSDataPrepAPIHelper.java]: there is a problem here on my local environment when test : RUNNING and NEW are in error, and seems to can't be resolved ?
maybe you should keep ?
AsyncExecution.Status.RUNNING || asyncStatus == AsyncExecution.Status.NEW

File: dataprep-test-api/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#backend.api.url=http://dev.data-prep.talend.lan:8888
+#backend.api.url=http://tdp-master-pe.datapwn.com
```
Comment (tlnd-jbrun) [inline dataprep-test-api/src/main/resources/application.properties]: this is a wanted modification ? if yes maybe we could keep the older value commented for developing purpose ?

Not related to this PR, but we also could add in the EE part amongst the commented lines, theses ones :
#maintenance/2.5
#backend.api.url=http://tal-rd44.talend.lan:9999/gateway 
?

File: dataprep-api/src/test/java/org/talend/dataprep/helper/OSDataPrepAPIHelper.java (hunk 1 above)
Comment (tlnd-jbrun) [reply dataprep-api/src/test/java/org/talend/dataprep/helper/OSDataPrepAPIHelper.java]: and  @throws InterruptedException seems to be useless...

File: dataprep-test-api/src/main/resources/application.properties (hunk 2 above)
Comment (aleboulanger) [reply dataprep-test-api/src/main/resources/application.properties]: `dev.data-prep.talend.lan:8888` no exist anymore

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/228/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/228/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/229/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/229/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/230/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/230/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/231/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/231/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/232/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/232/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/233/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/233/)

**Comment by aleboulanger** [comment]
@tlnd-jbrun build OK 👍

**Pull request merged**